bundles/automation/org.eclipse.smarthome.automation.integration.test/src/test/groovy/org/eclipse/smarthome/automation/integration/test/AutomationIntegrationJsonTest.groovy

```diff
@@ -249,7 +250,8 @@ class AutomationIntegrationJsonTest extends OSGiTest{
         def trigger = rule.triggers.find{it.id.equals("ItemStateChangeTriggerID")} as Trigger
         assertThat trigger, is(notNullValue())
         assertThat trigger.typeUID, is("core.GenericEventTrigger")
-        assertThat trigger.configuration.get("eventSource"), is ("myMotionItem")
+        //assertThat trigger.configuration.get("eventSource"), is ("myMotionItem") -> after bugfix
+        // the only way to check this is to run the rule and check that it is executed successfully
```

Comment:
I don't think we need the comment out line at all (if you add a note in the long description of the commit it should be enough if someone read the history to find the reason why the line has been removed).